modules/selenium/src/test/java/org/testcontainers/junit/Selenium3xTest.java

```diff
@@ -0,0 +1,33 @@
+package org.testcontainers.junit;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+/**
+ * Simple test to check that readiness detection works correctly across major versions of the containers.
+ */
+@RunWith(Parameterized.class)
+public class Selenium3xTest extends BaseWebDriverContainerTest {
```

Comment:
I think we can remove "extends BaseWebDriverContainerTest" because we don't use anything from it